src/components/Content/Auth/Signin.tsx

```diff
@@ -0,0 +1,101 @@
+import { Input, Button, FormProps, Form } from 'antd';
+import { authAPI } from '../../../api/authAPI';
+import { useEffect, useRef, useState } from 'react';
+import { AxiosError } from 'axios';
+
+type SigninFormType = {
+    login: string;
+    password: string;
+};
+
+export const Signin: React.FC = () => {
+    const [signinForm] = Form.useForm();
+    const [formError, setFormError] = useState({ enable: false, helpText: '' });
+    const formErrorRef = useRef(formError);
+
+    const onFinish: FormProps<SigninFormType>['onFinish'] = async (values) => {
+        try {
+            const response = await authAPI.signin(
```

Comment:
In previous components we used Redux to handle the logic and requests. The code looked complicated, so maybe we should discuss what is better way for that.